app/javascript/analytics/dashboard.js

```diff
@@ -44,8 +44,19 @@ function writeCards(data, timeRangeLabel) {
   followerCard.innerHTML = cardHTML(follows, `Followers ${timeRangeLabel}`);
 }
 
-function drawChart({ id, title, labels, datasets }) {
-  const options = {
+function drawChart({ id, timeRangeLabel, title, labels, datasets }) {
```

Comment:
I can see why it's been used, but I'm not convinced `timeRangeLabel` is the right thing to pass here, since we don't use it for anything other than deciding whether to hide the dots. Instead we could pass a boolean argument e.g. `showPoints` and change the below to e.g.

```js
  const chartOptions =
    showPoints
      ? {}
      : {
          elements: {
            point: {
              radius: 0,
            },
          },
        };
```

even better we could default that argument to `true` so that we only need to pass it in for the `Infinity` use case - e.g. 

```js
function drawChart({ id, showPoints = true, title, labels, datasets }) 
```

Reply:
Totally agree!